src/Magento/FunctionalTestingFramework/Util/TestGenerator.php

```diff
@@ -1303,6 +1303,8 @@ public function generateStepsPhp($actionObjects, $generationScope = TestGenerato
                 case "skipReadinessCheck":
                     $testSteps .= $this->wrapFunctionCall($actor, $actionObject, $customActionAttributes['state']);
                     break;
+                case "comment":
+                    $input = $input === null ? strtr($value, ['$' => '\$', '{' => '\{', '}' => '\}']) : $input;
```

Comment:
Add a comment as to why this line was added, not immediately clear what it's trying to do.